bin/hummingbot.py

```diff
@@ -78,7 +78,13 @@ async def main_async(client_config_map: ClientConfigAdapter):
 def main():
     chdir_to_data_directory()
     secrets_manager_cls = ETHKeyFileSecretManger
-    ev_loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
+
+    try:
+        ev_loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
+    except Exception:
```

Comment:
We should perhaps except on the particular exception here and in the quickstart one... I think this exception wouldn't actually be triggered on python 3.10 which we're using now, it's "future-proof" code for 3.11+